content/Voice Over - Fernanda Toledo.md

```diff
@@ -0,0 +1,63 @@
+# How to make your app accessible using Voice Over
+
+![](https://github.com/rootstrap/blog/blob/voice-over/content/images/glasses.jpg)
+
+In a world where everything is relying more and more on technology, more and more tools are being provided, for us developers to make our apps more accessible to everyone. Having limited sight should not be an impediment to use any app. As developers or designers, we need to raise awareness of all of those we're leaving behind and take a little time to make sure that our apps are accessible to everyone. Everyone wins, you'll broaden your market, increase your exposure, which will always lead to a positive impact on your profits. And the most important, by including people with disabilities you'll make a huge difference in their lives!
```

Comment:
`In a world where everything is relying more and more on technology, more and more tools are being provided`

This reads kind of funny to me, maybe avoid repeating `more and more`?

Reply:
You're totally right